build/usernameToColor-export.js

```diff
@@ -0,0 +1,7 @@
+const snapshot = require('./../src/functions/usernameToColor/__snapshots__/usernameToColor.spec.js.snap')
+const result = {}
+Object.keys(snapshot).map((key) => {
+		const uid = key.replace('usernameToColor ', '').replace(' has the proper color 1', '')
```

Comment:
There might be a more sane way to properly deserialize the jest snapshot but for now this does the trick